architecture/bela.cpp

```diff
@@ -731,6 +1058,8 @@ class BelaUI : public GenericUI
         EInOutPin fTrillPin;                        // current trill pin id
         vector<TrillWidget*> fTrillTable;           // list of TrillWidget
         const char* fTrillParams;
+        
+        vector<Trill*> gTouchSensors;        // List of Trill sensors
```

Comment:
Better `gTouchSensors` => `fTouchSensors`, "g" means  global and "f" means field

Reply:
ok. i make all the modification

Reply:
Have you pushed your modifications?

Reply:
no, i made there, but I can't test them right now. i think tomorrow. For now, i would like to leave the parsing code in bela.cpp